hypertrace-trace-enricher/hypertrace-trace-enricher-impl/src/main/java/org/hypertrace/traceenricher/enrichment/enrichers/resolver/backend/HttpBackendResolver.java

```diff
@@ -47,11 +52,13 @@
             EntityConstants.getValue(BackendAttribute.BACKEND_ATTRIBUTE_PATH),
             createAttributeValue(path));
       }
-      setAttributeForFirstExistingKey(
-          event, entityBuilder, HttpSemanticConventionUtils.getAttributeKeysForHttpMethod());
-      setAttributeForFirstExistingKey(
-          event, entityBuilder, HttpSemanticConventionUtils.getAttributeKeysForHttpRequestMethod());
-      return Optional.of(entityBuilder.build());
+      String attributeKey =
+          SpanAttributeUtils.getFirstAvailableStringAttribute(
+              event, HttpSemanticConventionUtils.getAttributeKeysForHttpMethod());
```

Comment:
You should be looking at both the lists `HttpSemanticConventionUtils.getAttributeKeysForHttpMethod()` and `HttpSemanticConventionUtils.getAttributeKeysForHttpRequestMethod()`

```
SpanAttributeUtils.getFirstAvailableStringAttribute(
        event,
        ImmutableList.copyOf(
            Iterables.concat(
                HttpSemanticConventionUtils.getAttributeKeysForHttpMethod(),
                HttpSemanticConventionUtils.getAttributeKeysForHttpMethod())));
```

Reply:
It is already taken care here while creating `event.getHttp()` - https://github.com/hypertrace/span-normalizer/blob/91667bf56c91a3d6beafee3a9edfd2fd7fc17e6f/span-normalizer/src/main/java/org/hypertrace/core/spannormalizer/fieldgenerators/HttpFieldsGenerator.java#L392

So, we can get an operation as below.
```
String operation = httpRequest.map(Request::getMethod).orElse(null);

```